lib/components/tree-node-collection.component.ts

```diff
@@ -51,10 +48,6 @@ export class TreeNodeCollectionComponent implements OnInit, OnDestroy {
     deprecatedSelector('TreeNodeCollection', 'tree-node-collection', elementRef);
   }
 
-  @action setNodes(nodes) {
```

Comment:
In MobX every change to state (observable) needs to occur inside an @action.
This is not forced by default, but it can be forced by calling mobx.useStrict(true).
The problem is - if someone else uses MobX in their app and they use strict mode, then this code will fail (I fixed an issue about it yesterday).

So these changes need to be reverted